main.py

```diff
@@ -29,6 +29,8 @@ async def websocket_endpoint(websocket: WebSocket):
                 pyautogui.moveRel(data['x'], data['y'])
             elif data['type'] == "tap":
                 pyautogui.leftClick()
+            elif data['type'] == "doubletap":
```

Comment:
nit: should we make it `double_tap` for readability?

Reply:
Actually it's consistent with hammer, similar to `tap`, so let's let it be.

Reply:
I agree. Just went back and played around with the semantics and it just _looked off_ during reading.